processing/data_collection/gazette/spiders/rn_natal.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+
+import dateparser
+from scrapy import Request, FormRequest
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RnNatalSpider(BaseGazetteSpider):
+
+    name = "rn_natal"
+    allowed_domains = ["www.natal.rn.gov.br"]
+
+    def start_requests(self):
+        base_url = "http://www.natal.rn.gov.br/dom/"
+        for year in range(2003, datetime.now().year + 1):
+            for month in range(1, 13):
+                data = dict(ano=str(year), mes=str(month).zfill(2), list="Listar")
+                yield FormRequest(url=base_url, formdata=data)
+
+    def parse(self, response):
+        for entry in response.css("#texto a"):
+            file_url = response.urljoin(entry.css("::attr(href)").get())
+            title = entry.css("::text").get()
+            date = dateparser.parse(title.split("-")[-1], languages=["pt"]).date()
+            extra_edition = "Extra" in title
+
+            yield Gazette(
+                date=date,
+                file_urls=[file_url],
+                is_extra_edition=extra_edition,
+                territory_id="2408102",
```

Comment:
I think you don't lose anything putting the `TERRITORY_ID` attribute on the spider, it makes it easier on the eye and makes it easier for refactoring and scripting. What do you think about it?

Reply:
I don't think this territory code should change. Therefore, I don't see any gain in moving it to a constant. If you really want me to do it, I can move it.

Reply:
I understand and agree that it shouldn't change much, at least for the capitals we can have a good amount of confidence that it will not change.

I think that making it a spider attribute also makes it easier for accessing it in a pipeline or something of sort (like allowing this field to be filled automatically if it's missing). This argument is not following YAGNI but I am trying to provide some reasons for "why to follow the project pattern and why it is good" and not just asking "just follow the project pattern".

I think an argument could also be made about not even needing the attribute since it can be read from the `territories.csv` file and with a bit of parsing we could accomplish the same result.

But, for now, I think we should follow the project pattern, for the sake of making it easier to find and some potential scripting, refactoring or adding of automatized pipeline elements.

Reply:
Moved to a constant.